apps/mudita-center-e2e/src/specs/overview/kompakt-overview.ts

```diff
@@ -42,16 +42,18 @@ describe("E2E mock sample - overview view", () => {
 
   it("Verify Overview Page", async () => {
     const kompaktImage = await OverviewKompaktPage.kompaktImage
-    await expect(kompaktImage).toBeDisplayed
+    await expect(kompaktImage).toBeDisplayed()
     console.log(kompaktImage)
     await expect(kompaktImage).toHaveAttribute("src", kompaktImageRegex)
 
     const kompaktOsVersion = await OverviewKompaktPage.kompaktOsVersion
-    await expect(kompaktOsVersion).toBeDisplayed
+    await expect(kompaktOsVersion).toBeDisplayed()
+    await expect(kompaktOsVersion).toHaveText("ANDROID 12")
```

Comment:
🔴  Testing failed  - MacOS 

I think you need to talk with devs, different value generation depending on the OS ? 

```
Error: Expect $(`[data-testid="version"]`) to have text

Expected: "ANDROID 12"
Received: "android 12"

```

Suggestion if no good answer from devs:
```suggestion
    const kompaktOsVersionText = await kompaktOsVersion.getText().toLowerCase();
    await expect(kompaktOsVersionText).toHaveText("android 12")
```

Reply:
I talked it with Łukasz, the exact case and on the newest build it is updated as "Mudita Os" is hardcoded, please check on 38 mock build

Reply:
Re-tested on mockdev.38 🟢